pkg/cmd/run/root.go

```diff
@@ -120,24 +123,27 @@ var runCmd = &cobra.Command{
 		}
 		tasklet.MustRun(startFunctions, tasklet.Opts{Signal: term})
 
-		fmt.Println("Local running, use ctrl-C to stop")
+		pterm.DefaultBasicText.Println("Local running, use ctrl-C to stop")
 
-		type apiendpoint struct {
-			Api      string `yaml:"api"`
-			Endpoint string `yaml:"endpoint"`
-		}
-		apis := []apiendpoint{}
+		stackState := run.NewStackState()
 
-		for a := range s.ApiDocs {
-			apis = append(apis, apiendpoint{Api: a, Endpoint: fmt.Sprintf("http://127.0.0.1:9001/apis/%s", a)})
-		}
-		if len(apis) == 0 {
-			// if we have a nitric.yaml then ApiDocs will be empty
-			for a := range s.Apis {
-				apis = append(apis, apiendpoint{Api: a, Endpoint: fmt.Sprintf("http://127.0.0.1:9001/apis/%s", a)})
-			}
-		}
-		output.Print(apis)
+		area, _ := pterm.DefaultArea.Start()
```

Comment:
we probably need something like
```
if s.LoadedFromFile() {
  //watch the file
} else {
  // do the stuff below
}
```

Reply:
I think with the new cli and our documentation we should deprecate yaml support. At least for now